examples/einsum.py

```diff
@@ -152,7 +161,58 @@ def run_einsum(expr, N, iters, dtype):
         help="number of times to benchmark this application (default 1 - "
         "normal execution)",
     )
+    parser.add_argument(
+        "--package",
+        dest="package",
+        type=str,
+        default="legate",
+        help="NumPy package to use (legate, numpy, or cupy)",
+    )
+    parser.add_argument(
+        "--cupy-allocator",
+        dest="cupy_allocator",
+        type=int,
+        default=0,
+        help="""cupy allocator to use (0-default memory pool,
+             1-turn off memory pool, 2-managed memory pool)""",
+    )
+    parser.add_argument(
+        "--cupy-compatibility",
+        dest="cupy_compatibility",
+        type=str,
+        default="false",
```

Comment:
Replace these two lines with `action="store_true"`. Then `args.cupy_compatibility` will be a boolean value directly, no need to check against the strings "true" and "false" below.

Reply:
Fixed. Thanks.